ftgo-common/src/main/java/net/chrisrichardson/ftgo/common/DeliveryActionType.java

```diff
@@ -0,0 +1,4 @@
+package net.chrisrichardson.ftgo.common;
```

Comment:
It doesn't make sense for this to be in `common`.
It's belongs in a `ftgo-delivery-service-*` module.